Packs/SailPointIdentityNow/Integrations/SailPointIdentityNowEventCollector/SailPointIdentityNowEventCollector.yml

```diff
@@ -0,0 +1,73 @@
+category: Analytics & SIEM
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: SailPointIdentityNowEventCollector - Dev
+  version: -1
+configuration:
+- display: IdentityNow Server URL (e.g. https://{tenant}.api.identitynow.com)
+  name: url
+  required: true
+  type: 0
+  section: Connect
+- display: Client ID
+  name: credentials
+  type: 9
+  required: true
+  displaypassword: Client Secret
+- defaultvalue: 50000
+  section: Collect
+  display: Max number of events per fetch
+  name: limit
+  required: false
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+  section: Connect
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+  section: Connect
+description: This is the SailPoint IdentityNow event collector integration for Cortex XSIAM.
+display: SailPoint IdentityNow Event Collector - Dev
+name: SailPointIdentityNowEventCollector - Dev
+supportlevelheader: xsoar
+script:
+  commands:
+  - arguments:
+    - auto: PREDEFINED
+      defaultValue: 'false'
+      description: If true, the command will create events, otherwise it will only display them.
+      name: should_push_events
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+    - description: Maximum number of results to return.
+      name: limit
+      required: false
+      defaultValue: 50
+    - description: An ID of the event to retrieve events from.
+      name: from_id
+      required: false
+    - default: false
+      description: Date from which to get events in the format of %Y-%m-%dT%H:%M:%SZ.
+      name: from_date
+    deprecated: false
+    description: Gets events from SailPoint IdentityNow.
```

Comment:
```suggestion
    description: Gets events from SailPoint IdentityNow. This command is used for developing/ debugging and is to be used with caution, as it can create events, leading to events duplication and API request limitation exceeding.
```